test/unit/lib/plugins/aws/package/compile/events/alb/index.test.js

```diff
@@ -219,4 +217,72 @@ describe('test/unit/lib/plugins/aws/package/compile/events/alb/index.test.js', (
         .and.have.property('code', 'ALB_NO_CONDITIONS');
     });
   });
+
+  describe('should support `functions[].events[].alb.targetGroupName` property', () => {
+    it('should use it if defined', async () => {
+      const result = await runServerless({
+        fixture: 'function',
+        command: 'package',
+        configExt: {
+          functions: {
+            fnTargetGroupName: {
+              handler: 'index.handler',
+              events: [
+                {
+                  alb: {
+                    ...validBaseEventConfig,
+                    priority: 1,
+                    targetGroupName: 'custom-targetgroup-name',
+                  },
```

Comment:
As `runServerless` runs are expensive, it'll be great to reuse one setup here: https://github.com/serverless/serverless/blob/3b1225a28abb08ac4d9d098e2f1606cf8b7c77b6/test/unit/lib/plugins/aws/package/compile/events/alb/index.test.js#L36.

Ideally we shoudl add there another function with `events[].alb.targetGroupName` and add:

```javascript
describe("general support", () => {
  it('should support `functions[].events[].alb.targetGroupName` property', () => {
    ...
  });
});
```

(it doesn't apply to second test, where we confirm on rejection, as those can only be tested with individual runServerless runs)

Reply:
Hello, @medikoo I'm not 100% sure how you want to organize the tests, there is no "general support section" currently.

So i kept my 2 tests in

* ```should support `functions[].events[].alb.targetGroupName` property```
  * ```should use it if defined```
  * ```should reject if `provider.alb.targetGroupPrefix` is also specified```

Do you wanted another organization ?